packages/std/src/testing/mock.rs

```diff
@@ -202,9 +202,12 @@ impl Api for MockApi {
         )?)
     }
 
+    #[cfg(feature = "std")]
     fn debug(&self, message: &str) {
         println!("{message}");
     }
+    #[cfg(not(feature = "std"))]
```

Comment:
Can we more the case distinction into the `fn debug` body? That avoids the repetition here